app/tests/test_browse.py

```diff
@@ -553,7 +562,9 @@ def test_browse_consignment_filter_display_multiple_pages(
     assert response.status_code == 200
     assert b"Search for digital records" in response.data
     assert b"You are viewing" in response.data
-    assert b"Everything available to you" in response.data
+    assert b"test body1" in response.data
```

Comment:
could we make the assertions clearer that these build up breadcrumbs. Currently this just assrts that this data is foundf anywhere on the page.

Can we assert the full breadcrumb html element is found and is as expected?

Similarlyl for all new breadcrumb assertions in the other tests

Reply:
sure, I can add new separate test cases for breadcrumbs

Reply:
I wasnt saying new test cases necessarily, just that these assertions of test bodt 1, testseries 1 etc dont show that they should be part of a breadcrum b structure. You can just replace these assertions you had added with one assertion per test changed like

assert b"<BREADCRUMB STRUCTURE>EVERYTHING,BODY,SERIES,ETC.</BREADCRUMB_STRUCTURE>" in response.data